README.md

```diff
@@ -35,7 +35,31 @@ docker pull ibmterraform/terraform-provider-ibm-docker
 
 ## Using the provider
 
-See the [IBM Provider documentation](https://ibm-cloud.github.io/tf-ibm-docs/) to get started using the IBM provider.
+If you want to run Terraform with the IBM Cloud provider plugin on your system, complete the following steps:
+
+1. [Download and install Terraform for your system](https://www.terraform.io/intro/getting-started/install.html). 
+
+2. [Download the IBM Cloud provider plugin for Terraform](https://github.com/IBM-Bluemix/terraform-provider-ibm/releases).
+
+3. Unzip the release archive to extract the plugin binary (`terraform-provider-ibm_vX.Y.Z`).
+
+4. Move the binary into the Terraform [plugins directory](https://www.terraform.io/docs/configuration/providers.html#third-party-plugins) for the platform.
+    - Linux/Unix/OS X: `~/.terraform.d/plugins`
+    - Windows: `%APPDATA%\terraform.d\plugins`
+
+5. Export the [IBM Cloud API key](https://cloud.ibm.com/iam#/users) as an environment variable.
+
+```sh
+export BM_API_KEY=API_KEY_VALUE
```

Comment:
We need these

    BM_API_KEY- The Bluemix API Key
    SL_API_KEY - The SoftLayer API Key
    SL_USERNAME - The SoftLayer username associated with the SoftLayer API Key.
We need above variables to be exported based on resource configuration(softlayer resources requires sl keys). May be you can add all three or line of description infrastructure resources requires sl keys